cmd/geth/main.go

```diff
@@ -261,6 +262,10 @@ func initGenesis(ctx *cli.Context) error {
 		utils.Fatalf("must supply path to genesis JSON file")
 	}
 
+	if ctx.GlobalBool(utils.TestNetFlag.Name) {
+		state.StartingNonce = 1048576 // (2**20)
+	}
```

Comment:
Do we need to update the nonces while just `init`-ing the genesis block? Doesn't only transaction processing depend on the nonce "trick"?